contributors/devel/sig-testing/monitoring.md

```diff
@@ -0,0 +1,212 @@
+
+Since these jobs run tests owned by other SIGs.
+Also make sure to periodically check on the "blocking" and "informing"
+dashboards for past releases.
+
+---
+
+## What do I do when I see a TestGrid alert?
+
+If you are part of a SIG's mailing list, occasionally you may see email from
+TestGrid reporting that a job or a test has recently failed.
+If you are casually browsing through TestGrid, you may also see jobs labeled as
+"flaky" (in purple) or as "failing" (in red).
+This section is to help guide you on what to do in these occasions.
+
+### Communicate your findings
+
+The number one thing to do is to communicate your finding (a test or job has
+been flaking or failing).
+If you saw a TestGrid alert on a mailing list, please reply to the thread and
+mentioned that you are looking into it.
+It is important to communicate to prevent duplicate work and to ensure CI
+problems get attention.
+
+In order to communicate with the rest of the community and to drive the work,
+please open up an issue on Kubernetes,
+https://github.com/kubernetes/kubernetes/issues/new/choose , chose the appropriate issue
+template.
+
+### Fill out the issue
+
+1. **Which job(s) are failing or flaking**
+
+The job is the tab in TestGrid that you are looking at.
+
+<img src="./testgrid-images/testgrid-jobs.png" height="50%" width="40%">
+
+The above example was taken from the SIG Release dashboard and we can see that
+* `conformance-ga-only` https://testgrid.k8s.io/sig-release-master-blocking#conformance-ga-only,
+* `skew-cluster-latest-kubectl-stable1-gce` https://testgrid.k8s.io/sig-release-master-blocking#skew-cluster-latest-kubectl-stable1-gce,
+* `gci-gce-ingress` https://testgrid.k8s.io/sig-release-master-blocking#gci-gce-ingress,
+* `kind-master-parallel` https://testgrid.k8s.io/sig-release-master-blocking#kind-master-parallel
+
+are flaky (we should have some issues opened up for these to investigate why
+:smile:).
+
+2. **Which tests are failing or flaking**
+
+Let's grab an example from the SIG release dashboards and look at the
+`node-kubelet-features-master` job in
+https://testgrid.k8s.io/sig-release-master-informing#node-kubelet-features-master.
+
+<img src="./testgrid-images/failes-tests.png" height="70%" width="100%">
+
+Here we see that at 16.07 EDT and 15:07 EDT the job
+```
+[k8s.io] NodeProblemDetector [NodeFeature:NodeProblemDetector] [k8s.io] SystemLogMonitor should generate node condition and events for corresponding errors [ubuntu]
+```
+Failed for Kubernetes commit `9af86e8db` (this value is a row below the time -
+the value above it is the run ID).
+The corresponding test-infra commit was `11cb57d36` (the value below the commit
+for Kubernetes).
+
+At 15:07 EDT, the job
+```
+[k8s.io] NodeProblemDetector [NodeFeature:NodeProblemDetector] [k8s.io] SystemLogMonitor should generate node condition and events for corresponding errors [cos-stable2]
+```
+
+failed as well.
+
+If one or both of these jobs continue failing, or if they failed frently
```

Comment:
Could we be more specific here? in the frequency of the failure, having an agreement can make it less subjective.